modules/logging/filewriter.go

```diff
@@ -33,6 +34,43 @@ func init() {
 	caddy.RegisterModule(FileWriter{})
 }
 
+// FileMode is a string made of 1 to 4 octal digits representing
+// a numeric mode as specified with the `chmod` unix command.
+// `"0777"` and `"777"` are thus equivalent values.
+type FileMode os.FileMode
```

Comment:
I wonder if we don't need to export this type either. :thinking: Save ourselves some API surface.

Reply:
Sure, i will unexport it and let someone else export it when needed.